RecoLocalFastTime/FTLRecProducers/python/mtdUncalibratedRecHits_cfi.py

```diff
@@ -1,17 +1,30 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoLocalFastTime.FTLCommonAlgos.mtdUncalibRecHitAlgo_cff import mtdUncalibRecHitAlgo
-
 from SimFastTiming.FastTimingCommon.mtdDigitizer_cfi import mtdDigitizer
 
-_barrelAlgo = mtdUncalibRecHitAlgo.clone()
-_barrelAlgo.adcNbits = mtdDigitizer.barrelDigitizer.ElectronicsSimulation.adcNbits
-_barrelAlgo.adcSaturation = mtdDigitizer.barrelDigitizer.ElectronicsSimulation.adcSaturation_MIP
-_barrelAlgo.toaLSB_ns = mtdDigitizer.barrelDigitizer.ElectronicsSimulation.toaLSB_ns
-_endcapAlgo = mtdUncalibRecHitAlgo.clone()
-_endcapAlgo.adcNbits = mtdDigitizer.endcapDigitizer.ElectronicsSimulation.adcNbits
-_endcapAlgo.adcSaturation = mtdDigitizer.endcapDigitizer.ElectronicsSimulation.adcSaturation_MIP
-_endcapAlgo.toaLSB_ns = mtdDigitizer.endcapDigitizer.ElectronicsSimulation.toaLSB_ns
+
+_barrelAlgo = cms.PSet(
+    algoName = cms.string("BTLUncalibRecHitAlgo"),
+    adcNbits = mtdDigitizer.barrelDigitizer.ElectronicsSimulation.adcNbits,
```

Comment:
it may be better to define these parameters in a common place and pick them up from that one place by the digitizer and reco.

What is the plan to put all these calibration constants in the DB?

Reply:
The plan is to put all the calibration constants in the DB. For what concerns the hardware specs, like the number of ADC bits, TDC bits, there range and so on, once the hardware is defined, I think I will follow the example of the other systems .